apps/sms/services/test/unit/messaging/messaging_service_test.js

```diff
@@ -0,0 +1,628 @@
+/*global bridge,
+         MocksHelper,
+         MockMessages,
+         Settings,
+         SMIL
+*/
+
+'use strict';
+
+require('/services/js/bridge_service_mixin.js');
+require('/services/test/unit/mock_bridge.js');
+require('/views/shared/test/unit/mock_messages.js');
+require('/views/shared/test/unit/mock_settings.js');
+require('/views/shared/test/unit/mock_smil.js');
+
+var MocksHelperForAttachment = new MocksHelper([
+  'bridge',
+  'Settings',
+  'SMIL'
+]).init();
+
+suite('Messaging service >', function() {
+  var MessagingService, successResult, errorResult;
+
+  MocksHelperForAttachment.attachTestHelpers();
+
+  suiteSetup(function(done) {
```

Comment:
mmm, I remember we agreed to get rid of ```suiteSetup``` and ```suiteTeardown``` enterily and move ```require('/services/js/messaging/messaging_service.js')``` to the place where the rest of ```require``` is located? :)